toolbox/library/src/OSQP.cpp

```diff
@@ -298,7 +305,7 @@ bool wbt::block::OSQP::solverInitialization(const BlockInformation*blockInfo)
     if (pImpl->useLb || pImpl->useUb) {
         pImpl->numberOfTotalConstraints += pImpl->numberOfVariables;
     }
-
+    
```

Comment:
Nitpick: do not add trailing whitespace (see https://stackoverflow.com/questions/21410075/what-is-trailing-whitespace-and-how-can-i-handle-this).